module/files/helper.js

```diff
@@ -1,126 +1,403 @@
 /**
- * name : files.js
- * author : Vishnu
- * created-date : 01-Sep-2023
- * Description :  Files helper.
-*/
+ * name : files/helper.js
+ * author : prajwal
+ * created-date : 25-Apr-2024
+ * Description : All files related helper functionality.Including uploading file
+ * to cloud service.
+ */
 
-const cloudServices = require(GENERICS_FILES_PATH + "/services/cloudStorage")
+// Dependencies
+const Zip = require("adm-zip");
+const fs = require("fs");
+const { cloudClient } = require(PROJECT_ROOT_DIRECTORY + "/config/cloud-service");
+let cloudStorage = process.env.CLOUD_STORAGE_PROVIDER;
+
+/**
+ * FilesHelper
+ * @class
+ */
 
 module.exports = class FilesHelper {
+  /**
+   * Get downloadable url
+   * @method
+   * @name                        - getDownloadableUrl
+   * @param {filePath}            - File path.
+   * @param {String}              - Bucket name
+   * @param {Array} storageName   - Storage name if provided.
+   * @param {Number} expireIn     - Link expire time.
+   * @return {String}             - Downloadable url link
+   */
+
+  static getDownloadableUrl(
+    filePath,
+    bucketName,
+    storageName = "",
+    expireIn = ""
+  ) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let noOfMinutes = CONSTANTS.common.NO_OF_MINUTES;
+        let linkExpireTime = CONSTANTS.common.NO_OF_EXPIRY_TIME * noOfMinutes;
```

Comment:
@Prajwal17Tunerlabs make use of env variables

Reply:
using env variable instead of common.js